### PR TITLE
Add "DTLSconnector thread count" property to the DTLSConnectorConfig.

### PR DESCRIPTION
**Pull request by sbernard31**

### CODE REVIEW

File: scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java [hunk 1]
```diff
@@ -340,9 +340,12 @@ private void start(final InetSocketAddress bindAddress) throws IOException {
 				new DaemonThreadFactory("DTLS RetransmitTask-", NamedThreadFactory.SCANDIUM_THREAD_GROUP));
 
 		if (executor == null) {
-			// use a decently sized thread pool
-			executor = new StripedExecutorService(DEFAULT_EXECUTOR_THREAD_POOL_SIZE);
-			this.hasInternalExecutor = true;
+			if (config.getConnectionThreadCount() == null) {
+				executor = new StripedExecutorService(DEFAULT_EXECUTOR_THREAD_POOL_SIZE);
+			} else {
```
Comment (boaks) [inline scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: `this.hasInternalExecutor = true;`

missing.

Comment (boaks) [inline scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: in 344

Comment (sbernard31) [inline scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: Thx :)

**Comment by boaks** [comment]
LGTM

**Pull request merged**